Login_SpotifyUI/Login_SpotifyUI/Controller/LoginViewController.swift

```diff
@@ -8,8 +8,16 @@
 import UIKit
 import SnapKit
 
+enum HidePasswordState {
+    case hide
+    case open
+}
+
 class LoginViewController: UIViewController {
     let loginView = LoginView()
+    var passwordData: String = ""
+    var hidePasswordData: String = ""
+    var normalPasswordState = HidePasswordState.hide
```

Comment:
`normal`이라는 prefix를 붙이지 않아도 충분히 의미 전달이 가능할 것 같습니다~!
```suggestion
    var passwordState = HidePasswordState.hide
```